packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -9,17 +9,16 @@ import type {
   GetRelease,
   Release,
   DeleteRelease,
-  GetContentTypeEntryReleases,
   GetReleases,
   MapEntriesToReleases,
 } from '../../../shared/contracts/releases';
 import type { UserInfo } from '../../../shared/types';
-import { getService } from '../utils';
+import { getService, getEntryId } from '../utils';
 
 type ReleaseWithPopulatedActions = Release & { actions: { count: number } };
 
 const releaseController = {
-  async findMany(ctx: Koa.Context) {
+  async findByDocumentAttached(ctx: Koa.Context) {
```

Comment:
could you add some comment or explanation of what's the responsability of this controller? 🤔 
After 2 weeks of being of, I forgot a bit the logic, and I don't quite see what `findByDocumentAttached`  means.

Maybe what confuses me is the "Attached" word

Reply:
Also, it feels off that the endpoint is called findByDocumentAttached, but you can send a parameter to find by documents not attached 🤔

Reply:
I think it's a bit confusing for me given the name of the controller, calling a findByDocumentAttached, sending a `hasEntryAttached: false` looks odd.

Also, its mixing document & entry language in here which makes it a bit more complicated imo